src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="headerNav">
+      <a href="#home">
+        <img 
+          class="navIcon" 
+          src="images/logo.png" 
+          alt="icon"
+        >
+      </a>
+      <nav class="navBlock">
+        <ul class="navList">
+          <li class="navItem">
+            <a class="navLink is-active" href="#apple">Apple</a>
```

Comment:
To get clear code, you should decide what type of cases use all over the project for naming classes. 
Here you have used as kebab-case, as camelCase.
So, please rewrite all classes using ONE case.